configure.ac file  has typo

At 334 line, there are typo
AS_IF([test "${prefix}" = "NONE"],[AS_VAR_SET([prefix],[${ac_defaut_prefix}])]);

I think 'ac_defaut_preifx'  is typo.
the correct name is ac_default_prefix.
because of this typo, MDSplus does not configured well.
it keep failed at some configure line. 
it did not compile and link the library because generated command was wrong.
the CLAGS of MacOS is 

  CFLAGS="$CFLAGS -fsigned-char -fno-strict-aliasing -Wl,-rpath,@loader_path/..,-rpath,@executable_path/..,-rpath,$exec_prefix";

After including path at CFLAGS variables, it include @exec_prefix, and this valuable set to prefix if exec_prefix is None and exec_prefix is set ac_default_prefix if prefix is None. but as I already mentioned there are typo 'ac_defaut_prefix'.therefore it can not bring the right value it remain empty and it occurs the following error.

error message when do configure.

configure:8695: gcc -o conftest -O3 -fvisibility=hidden  -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -fsigned-char -fno-strict-aliasing -Wl,-rpath,@loader_path/..,-rpath,@executable_path/..,-rpath,  -I${top_builddir}/_include -I${top_srcdir}/_include -I${top_builddir}/include -I${top_srcdir}/include  -L/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/lib conftest.c  >&5
Undefined symbols for architecture x86_64:
  "_main", referenced from:
     implicit entry/start for main executable
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
configure:8695: $? = 1
configure: program exited with status 1
configure: failed program was:

--- a/configure.ac
+++ b/configure.ac
@@ -331,7 +331,7 @@ else
   MAKEFLAG_J="-j -l 1"
 fi
 
-AS_IF([test "${prefix}" = "NONE"],[AS_VAR_SET([prefix],[${ac_defaut_prefix}])])
+AS_IF([test "${prefix}" = "NONE"],[AS_VAR_SET([prefix],[${ac_default_prefix}])])
 AS_IF([test "${exec_prefix}" = "NONE"],[AS_VAR_SET([exec_prefix],[${prefix}])])
 
 MAKEFLAG_O="$(if ${MAKE:=make} -v -O&>/dev/null; then echo -O; fi)"

